src/formats/roam-json.ts

```diff
@@ -86,7 +112,41 @@ export class RoamJSONImporter extends FormatImporter {
 				}
 				const filename = `${graphFolder}/${pageName}.md`;
 
-				const markdownOutput = await this.jsonToMarkdown(graphFolder, attachmentsFolder, pageData);
+				// if title option is enabled
+				let YAMLtitle: string;
```

Comment:
We're already passing pageData into `jsonToMarkdown`. Can we extract this value in that function so we don't need to pass it as a parameter?

Reply:
I wasn't sure how to access it through the function, when I tried to pull the `title` it didn't show available unless I got it directly from `pageData` beforehand. Is it because `jsonToMarkdown` can get either a page _or_ a block?

Trying to use `json.title` shows "Property 'title' does not exist on type 'RoamPage | RoamBlock'." Agreed, if there's another way to access it then I'd rather do it through `jsonToMarkdown`